lib/facter/monit_version.rb

```diff
@@ -0,0 +1,14 @@
+# A facter fact to determine the installed version of monit.
+
+module Facter::Util::MonitVersion
+  class << self
+    def get_monit_version
+      monit_version = Facter::Util::Resolution.exec('monit -V 2>&1')
```

Comment:
@ghoneycutt There's some reason but I can't remember offhand, I was asked to add it in when I added git and java facts to the PuppetLabs modules. I'll see if I can find out why :+1: